rts/Game/UI/MiniMap.cpp

```diff
@@ -565,6 +563,26 @@ void CMiniMap::SelectUnits(int x, int y)
 
 /******************************************************************************/
 
+void CMiniMap::MouseWheel(bool up, float delta)
+{
+	float3 mapPos = GetMapPosition(mouse->lastx, mouse->lasty);
+	mapPos.y = CGround::GetHeightAboveWater(mapPos.x, mapPos.z, false);
+
+	float3 screenMousePos = camera->CalcViewPortCoordinates(mapPos);
+
+	// If cursor position in minimap refers to a point outside camera view just move to it
+	if (screenMousePos.x < 0 || screenMousePos.x > globalRendering->viewSizeX ||
```

Comment:
It would be nice if camera had a function to test for a map position being contained by current view, however this is good enough.

Reply:
Done